src/test/java/serviceimpl/FileWriterImplTest.java

```diff
@@ -0,0 +1,55 @@
+package serviceimpl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.FileWriter;
+
+public class FileWriterImplTest {
+    private static FileWriter fileWriter;
+    private static final String CORRECT_REPORT_PATH = "src/main/resources/CloseDay.csv";
+    private static final String WRONG_REPORT_PATH = "src/main/resources/CloseDay.csv";
+    private static String expectedString;
+
+    @BeforeClass
+    public static void beforeClass() {
+        expectedString = "fruits, quantity" + System.lineSeparator()
+                + "banana" + "," + "152" + System.lineSeparator()
+                + "apple" + "," + "90";
+    }
+
+    @Test
+    public void fileWriter_Ok() {
+        String expected = expectedString;
+        String actual = getReadFile(WRONG_REPORT_PATH);
+        Assert.assertEquals(expected, actual);
```

Comment:
you are not writing to file in this method
write to file with your service, than read from that file with reader from io or nio, and check expected and actual